packages/payment-detection/src/eth/fee-proxy-contract.ts

```diff
@@ -0,0 +1,334 @@
+import { ethereumFeeProxyArtifact } from '@requestnetwork/smart-contracts';
```

Comment:
this class is 90% identical to others... it's time for a refactoring don't you think?

Reply:
Discussed IRL. it will be my next priority. So let's keep it for a next PR.